UI/src/app/components/pages/mapping/mapping.component.ts

```diff
@@ -14,17 +17,29 @@ export class MappingComponent implements OnInit {
   common$: Observable<any>;
   data$: Observable<any>;
 
-  constructor( private store: Store<any>, private dataService: DataService ) {
-      this.common$ = store.pipe(select('common'));
-      this.data$ = store.pipe(select('data'));
+  constructor(
+    private store: Store<any>,
+    private dataService: DataService,
+    @Inject(DOCUMENT) private document: Document,
+    private httpClient: HttpClient
+  ) {
+    this.common$ = store.pipe(select('common'));
+    this.data$ = store.pipe(select('data'));
   }
 
   ngOnInit() {
-    this.store.dispatch(new Collapsed());
-    this.store.dispatch(new dataActions.FetchData())
+    // this.httpClient.get<any>(`http://127.0.0.1:5000/load_report?path=C:/Projects/CDM/CDMSouffleur/UI/mdcr.xlsx`)
```

Comment:
don't commit to PR commented unworking code (also this part of changes doesn't belong to the refactoring )